test/testtokenize.cpp

```diff
@@ -994,6 +995,15 @@ class TestTokenizer : public TestFixture {
         ASSERT_EQUALS(";\n\nasm ( \"\"mov ax,bx\"\" ) ;", tokenizeAndStringify(";\n\n__asm__ volatile ( \"mov ax,bx\" );"));
     }
 
+    void simplifyAsm() {
+        // asm label (variable)
+        ASSERT_EQUALS("int * pfoo ; pfoo = NULL ;",
+                      tokenizeAndStringify("int* pfoo asm (\"pmyfoo\") = NULL;"));
```

Comment:
this simplification is OK

Reply:
Sorry, do you mean that is it just OK, or that this check is redundant? I've added this test because current simplification doesn't look right:
`1: int * pfoo asm@var1 ( ""pmyfoo"" ) ; = NULL ;`